### PR TITLE
feat: payment

### PR DESCRIPTION
**Pull request by nayyyhaa**
## In this PR :
-  I've added Razorpay integration  +few UI fixes

#### Component files

- [x] [components/Card.js](https://github.com/nayyyhaa/inNotion-care-react/compare/payment-branch?expand=1&short_path=b335630#diff-7916963985f6b54523951aa9bdcdb855db5905bed40fc7775a485986aec0fa5d)
- [x] [components/CartSidebar.js](https://github.com/nayyyhaa/inNotion-care-react/compare/payment-branch?expand=1&short_path=b335630#diff-bc3b67a94629380a52356fc4046c3c92e564e1634d523d9a0620c6c53d76c31c)

#### Page
- [x] [pages/CheckoutPage.js](https://github.com/nayyyhaa/inNotion-care-react/compare/payment-branch?expand=1&short_path=b335630#diff-8c5148e4fe86886ce753a8cedea65f09a41a1c5a7912363f2866d41575dfc3be)
- [x] [pages/OrderPage.js](https://github.com/nayyyhaa/inNotion-care-react/compare/payment-branch?expand=1&short_path=b335630#diff-22c8c76ac6d2b2ab1af16bcca235debc418ec2da824071766f90dbd3ded3b1ce)

#### utils:
- [x] [contexts/OrderContext.js](https://github.com/nayyyhaa/inNotion-care-react/compare/payment-branch?expand=1&short_path=b335630#diff-fb594e05ea1de315c5a3d8f95847b688703854978bcbaa0d1912465821121a29)
 
#### files to *ignore*
- [ ] files with just few import changed
- [ ] CSS files

### Live preview : https://deploy-preview-8--innotion-carev2.netlify.app/

### CODE REVIEW

File: src/components/CartSidebar.js [hunk 1]
```diff
@@ -19,6 +29,80 @@ export const CartSidebar = ({ totalPrice, finalPrice, isShippingFree }) => {
...
+
+      var options = {
+        key: "rzp_test_hTxMgISWpX1WzU",
+        amount: finalPrice * 100,
+        currency: "INR",
+        name: "inNotion Care",
+        description: "Test Transaction",
+        image: compPhoto,
+        handler: function (response) {
+          //  alert(response.razorpay_payment_id);
```
Comment (hsnice16) [inline src/components/CartSidebar.js]: Check if these commented code are not required, you can remove.

**Review by hsnice16 — APPROVED** [review]
Nice work. Clean code. Couldn't find much to suggest. Good Job.

**Review by Developers-codz — APPROVED** [review]
The code looks Clean couldn't found anything to suggest you can merge

File: src/components/CartSidebar.js (hunk 1 above)
Comment (nayyyhaa) [reply src/components/CartSidebar.js]: Made the changes.

**Comment by nayyyhaa** [comment]
Thanks all for reviewing. Merging the changes.

**Pull request merged**